# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../heimdal/slapd-module-smbk5pwd-hotfix.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./contrib/slapd-modules/smbk5pwd/Makefile.orig	2005-02-28 18:00:12.000000000 -0300
+++ ./contrib/slapd-modules/smbk5pwd/Makefile	2005-02-28 18:05:56.587166064 -0300
@@ -35,3 +35,8 @@
 smbk5pwd.la:	smbk5pwd.lo
 	$(LIBTOOL) --mode=link $(CC) $(OPT) -version-info 0:0:0 \
 	-rpath /usr/local/libexec/openldap -module -o $@ $? $(LIBS)
+
+smbk5pwd.so:	smbk5pwd.o
+	$(CC) -shared -Wl -o smbk5pwd.so smbk5pwd.o
+
+
